sites/public/src/components/account/ApplicationsView.tsx

```diff
@@ -0,0 +1,230 @@
+  All = 0,
+  Lottery,
+  Open,
+  Closed,
+}
+interface ApplicationsCount {
+  total: number
+  lottery: number
+  open: number
+  closed: number
+}
+
+interface ApplicationsViewProps {
+  filterType: ApplicationsFilterEnum
+}
+
+const ApplicationsView = (props: ApplicationsViewProps) => {
+  const { applicationsService, listingsService, profile } = useContext(AuthContext)
+  const [applications, setApplications] = useState<AppWithListing[]>()
+  const [applicationsCount, setApplicationsCount] = useState<ApplicationsCount>()
+  const [loading, setLoading] = useState(true)
+  const [listLoading, setListLoading] = useState(true)
+  const [error, setError] = useState()
+  const router = useRouter()
+
+  useEffect(() => {
+    if (profile && loading) {
+      pushGtmEvent<PageView>({
+        event: "pageView",
+        pageTitle: `My Applications - ${ApplicationsFilterEnum[props.filterType]}`,
+        status: UserStatus.LoggedIn,
+      })
+      applicationsService
+        .list({ userId: profile.id })
+        .then((apps) => {
+          apps?.items?.length > 0 ? setApplications(apps.items) : setLoading(false)
+        })
+        .catch((err) => {
+          console.error(`Error fetching applications: ${err}`)
+          setError(err)
+          setLoading(false)
+        })
+    }
+  }, [profile, applicationsService])
+
+  useEffect(() => {
+    if (!applications || (applications && !listLoading)) return
+
+    void Promise.all(
+      applications?.map(async (app) => {
+        const retrievedListing = await listingsService.retrieve({
+          id: app?.listings.id,
+        })
+        app.fullListing = retrievedListing
+        return app
+      })
+    )
+      .then((res) => {
+        setListLoading(false)
+        const displayApplications = []
+        const total = res.length
+        let open = 0,
+          closed = 0,
+          lottery = 0
+        res.forEach((app) => {
+          if (app.fullListing?.status === ListingsStatusEnum.active) {
+            open++
+            if (props.filterType === ApplicationsFilterEnum.Open) displayApplications.push(app)
+          } else if (app.fullListing?.lotteryStatus === LotteryStatusEnum.publishedToPublic) {
+            lottery++
+            if (props.filterType === ApplicationsFilterEnum.Lottery) displayApplications.push(app)
+          } else {
+            closed++
+            if (props.filterType === ApplicationsFilterEnum.Closed) displayApplications.push(app)
+          }
+        })
+        props.filterType === ApplicationsFilterEnum.All
+          ? setApplications(res)
+          : setApplications(displayApplications)
+        setApplicationsCount({ total, lottery, open, closed })
+        setLoading(false)
+      })
+      .catch((err) => {
+        console.error(`Error fetching applications: ${err}`)
+        setError(err)
+        setLoading(false)
+      })
+  }, [applications, listLoading, listingsService])
+
+  const selectionHandler = (index: number) => {
```

Comment:
FWIW, this definitely leads me to believe that what we really need is the to-be-developed Seeds Menu component. The Tabs component is intended to switch views already loaded on-page without changing routes. I'm not sure if the a11y on this quite makes sense.

Reply:
To add on to this, do we need to fully change routes when switching tabs? In the past when I've worked on a page like this I used a context to store the data so when switching between tabs I just grab the appropriate data. It's fairly performant, but having the whole page load when you click on any of the tabs on the left side makes the page feel a little slower and jarring than it needs to be.

And if we need to keep track of what tab is selected in the URL we could use a query param rather than a path param.